src/crypto-providers/ledgerCryptoProvider.ts

```diff
@@ -279,8 +336,9 @@ export const LedgerCryptoProvider: () => Promise<CryptoProvider> = async () => {
       ttl,
       certificates,
       withdrawals,
+      undefined,
```

Comment:
I'd rather pass null than undefined, I find it more "assertive" of the fact that the given property is missing

Reply:
Fixed